### PR TITLE
Change the background image of the search page header

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/5676

image

### CODE REVIEW

File: pages/search.js [hunk 1]
```diff
@@ -300,7 +300,8 @@ class SearchPage extends React.Component {
     return (
       <Page title="Search" showSearch={false}>
         <Container
-          backgroundImage="url(/static/images/search-background.png)"
+          backgroundImage="url(/static/images/home/fiscalhost-blue-bg-lg.png)"
+          style={{ transform: 'rotate(180deg)' }}
```
Comment (znarf) [inline pages/search.js:304]: What is the purpose of the rotation there?

Comment (SudharakaP) [reply pages/search.js:304]: Note that this image originally was intended for the homepage,

image

and as such it is inverted. We need to rotate it by 180 degrees for it to be used in the `/search` page.

Comment (znarf) [reply pages/search.js:304]: I find that approach a bit hacky but ok.

**Review by znarf — APPROVED** [review]

File: pages/search.js (hunk 1 above)
Comment (SudharakaP) [reply pages/search.js:304]: Alternative maybe is to store a duplicate of this image in the correct rotation set. But I didn't do that as that introduces duplication of a file unnecessary. 🤔

**Pull request merged**